### ISSUE TITLE
valid_sub_YouTube_VOS_dataset

### ISSUE THREAD

**Issue by qihao-huang** [MinghanLi/STMask#5]
```
valid_sub_YouTube_VOS_dataset = dataset_base.copy({
    'img_prefix': '../datasets/YouTube_VOS2019/train/JPEGImages',
    'ann_file': '../datasets/YouTube_VOS2019/annotations_instances/valid_sub.json',
    'test_mode': False,
})
```
@MinghanLi 
It sounds like you split the train jsons into `train_sub` and `valid_sub` for epoch/iterations evaluation. 
Could you provide more details about this one such as how to split to reproduce your results? I could not find it across the codes.

And, is that the `train-sub` and `valid_sub`are only used in developments, and all reported results are trained by full train set after 12 epochs for YT and 20 epochs for OVIS?

Thanks!

**Comment by MinghanLi** [comment]
Since the GT of validation set is not public,  we have to submit the .json file of validation set  to the official web of Codelab to test the performance of models. It is very inconvenient.  We, therefore, divide the train set of train.json into train_sub.json (top nine tenths) and valid_sub.json (the last tenth), hoping to make it more convenient to verify some ideas. 

But for all reported results, we keep same settings with other methods. We train STMask on all train set after 12 epochs for YT and 20 epochs for OVIS. Because the more data, the better the result.
